app/services/v1/sms/replay.rb

```diff
@@ -33,4 +46,71 @@ def delete_messaged_for(number)
       job.delete if job_args['job_class'] == 'SmsPlans::SendSmsJob' && job_args['arguments'].first.eql?(number)
     end
   end
+
+  def handle_message_with_sms_code
+    session = Session::Sms.includes(:intervention).where(intervention: { status: :draft }).find_by(sms_code: message)
```

Comment:
Why are we looking for a `draft` session?

Reply:
Committed by accident 😶